tasks/internal_reconcile_report_phantom/schemas/output.json

```diff
@@ -0,0 +1,52 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/internal_reconcile_report_phantom/schemas/output.json",
+    "title": "internal_reconcile_report_phantom Lambda Output",
+    "description": "The output for the internal_reconcile_report_phantom Lambda.",
+    "type": "object",
+    "required": [
```

Comment:
I think this may need to be worked a little as I believe the output would be an array of objects. Verify with @AndrewDorn but I believe you would something like the following:
```
{
  jobId: integer,
  pageNumber: Integer,
  nextPage: boolean,
  phantoms: [
    {
      collectionId: string,
      granuleId: string,
      ...
    },
    {
      ...
    }
  ]
}
```

We also may want to include `createdAt` time from cumulus maybe and `cumulusBucket` thoughts?

Reply:
`createdAt `and `cumulusBucket` are not needed for now.